### DISCUSSION TITLE
Blocked - but not by a Task?

### DISCUSSION THREAD

**Discussion by skostojohn** [GothenburgBitFactory/taskwarrior#2497] [Q&A] [answered]
I am curious what the best way is to manage tasks that are blocked, but not by another task.

For example, I need to do some work, so I create a task to get it out of my head and onto my list.  But I can't work on it until someone else takes a decision.  So I want to mark it as "blocked" so it is color coded as blocked, and it drops off my ready list, but there is no task for me that is blocking it - so nothing to set via depends:.

I can use regular tags to mark these tasks, but then they still show up as "ready" even though they are not currently actionable.

Any ideas?

Thanks!!
Scott

**Comment by duckunix** [comment, marked as answer]
Update your ```next``` report with a filter and add a custom tag. For example, I have:

```
report.next.filter     =status:pending limit:page -BLOCKED -someday
```
Just add ```-MYBLOCKED``` in the filter line.  No need to overwrite the rest of the report.

**Reply by tbabej** [reply]
@duckunix Great approach!

I would just add that in order to really have them equivalent to normally blocked tasks, you might want to consider creating a custom rule to bump the urgency value for the tasks having the `myblocked` tag, i.e. putting something like

```
urgency.user.tag.myblocked.coefficient=-5.0
```

in your `taskrc` (-5.0 is the default value). Similarly you can address the coloring by setting:

```
color.tag.myblocked=color0 on color10
```

**Comment by skostojohn** [comment]
@duckunix, @tbabej  - these are great suggestions - thank you so much for your help!

**Comment by djmitche** [comment]
I realize this is answered, but I'll mention my approach too: I mark such tasks as waiting -- for a few hours, for a day, for a week, as the needs dictate.  I usually also add an annotation when I do so.  For example, if I'm planning a trip, I might have a a "rent kayaks" task.  I might call the kayak place and leave a message, then give them a day to return my call:

```
task 123 annotate called abc kayaks and left a message wait:1d
```

then when the task reappears, either it's done (they called me back) or it's time to follow up on it.

**Reply by jaker-dotcom** [reply]
Exactly my approach. Simpler than than the solutions above and it doesn't take much time to set new wait date if the condition isn't met yet. Also when it shows up one may remind the person that is responsible for the wait that one is indeed waiting.
